internal/config/system.go

```diff
@@ -62,6 +62,9 @@ func (s *System) DefaultAndValidate() error {
 	if s.ModelAutoscaling.TimeWindow.Duration == 0 {
 		s.ModelAutoscaling.TimeWindow.Duration = 10 * time.Minute
 	}
+	if s.ModelAutoscaling.StateConfigMapName == "" {
+		s.ModelAutoscaling.StateConfigMapName = "kubeai-autoscaler-state"
```

Comment:
I think we should remove that check and fail if it's not set, since it should always be set by the default value from helm.

Reply:
Done